build.gradle

```diff
@@ -218,6 +218,41 @@ ext {
     log4JVersion = '2.17.1'
 }
 
+dependencyManagement {
+    dependencies {
+        dependencySet(group: 'io.netty', version: versions.netty) {
```

Comment:
I don't think we need versions.netty anymore (it is defined above this code). You can replace it with the actual version number:
[       dependencySet(group: 'io.netty', version: '4.1.74.Final') {](url)

Reply:
Updated.